web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -193,12 +200,23 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
       if (!alreadyExists) {
         handleOpen();
         const res = await createReviewPeriod(periodToAdd, csrf);
-        const data =
-          res && res.payload && res.payload.data ? res.payload.data : null;
-        data && dispatch({ type: ADD_REVIEW_PERIOD, payload: data });
+        const data = res?.payload?.data ?? null;
+        console.log('Data');
+        console.log(data);
+        if (data) {
+          dispatch({ type: ADD_REVIEW_PERIOD, payload: data });
+        } else {
+          console.log(res?.error);
```

Comment:
I imagine we need better error handling here like display a message in a dialog.
I'm not familiar with how other similar errors are handled in Check-ins.

Reply:
Added a toast for the error to update the user on what went wrong.

Reply:
Nice! If you also want to keep the console output, I would change it to use `console.error` instead of `console.log`. That will make it appear in red.